Problem_1_HashMap.py

```diff
@@ -0,0 +1,21 @@
+class Problem_1:
+	def findContinuousArray(self, arr, target):
+		runningSumDict = dict()
+		count = 0
+		runningSumDict[0] = 1
+		runningSum = 0
+		for i in arr:
+			runningSum += i
+			compliment = runningSum - target
```

Comment:
it should be target-runningsum

Reply:
It has to be runningSum - target right? Because the running sum keeps on increasing and it will be greater than target. And as far as I remember thats how the algorithm works. In the class, the examples always had the positive compliment to be checked in the hashmap